ocp_resources/resource.py

```diff
@@ -654,13 +655,21 @@ def update_replace(self, resource_dict):
         self.api.replace(body=resource_dict, name=self.name, namespace=self.namespace)
 
     @staticmethod
-    def _retry_cluster_exceptions(func):
+    def _retry_cluster_exceptions(
+        func,
```

Comment:
Any reason why you have here multiple lines?

Reply:
I thought this was done by "pre-commit", I did not create it like this intentionally, let me try to see if I can undo this.